bigtable-client-core-parent/bigtable-client-core/src/main/java/com/google/cloud/bigtable/grpc/BigtableSession.java

```diff
@@ -99,6 +103,8 @@
   private static final Logger LOG = new Logger(BigtableSession.class);
   // TODO: Consider caching channel pools per instance.
   private static ManagedChannel cachedDataChannelPool;
+  // Hash map containing cached connections to specific destination hosts for GCJ client
+  private static HashMap<String, ClientContext> gcjClientContext = new HashMap<>();
```

Comment:
Also please rename the variable to something like `cachedGcjClientContexts` to indicate that it contains many contexts

Reply:
I don't want to block this PR on this, but in the future we should make the ClientContexts reference counted, so that when BigtableSession.close() is called we can start cleaning them up. Would you mind adding a `// TODO: ` comment?